src/client/components/helpers.ts

```diff
@@ -0,0 +1 @@
+export const isLocalLink = (link: string) => /^\/(?!\/)/.test(link);
```

Comment:
Maybe it should be in `/src/shared/utils/link.ts`?

Reply:
Yep, I was trying to replicate fingerprint.com folder structure to make importing easier but only ended up using this one utility in the end. Moved to `src/shared/utils` as suggested.